docs/migrating_to_17.md

```diff
@@ -84,6 +84,21 @@ bento_authz public-data-access counts
 ```
 
 
+## 5. Optionally, add Beacon network
+
+To host a network of beacons, with a corresponding UI in bento_public, first copy the config file: 
+
+```bash
+./bentoctl.bash init-config beacon-network
+```
+
+then update values at `lib/beacon/config/beacon_network_config.json`. Activate the network by adding (or modifying) this value in local.env:
+
+
+```bash
+BENTO_BEACON_NETWORK_ENABLED='true'
+```
+
 ## 5. Start Bento
```

Comment:
make this 6

Reply:
They're so closely tied I didn't see the point

Reply:
even still, I don't think it makes sense to have two step 5s, with one of them being optional and the other being required (and coming after the first step 5)

Reply:
right, I was thinking of the env step as optional also, but I guess it needs to be throw in regardless